prbt_hardware_support/README.md

```diff
@@ -64,3 +64,7 @@ The ``ModbusAdapterBrakeTestNode`` is noticed via the topic `/pilz_modbus_node/m
 ## BraketestExecutorNode
 The ``BraketestExecutorNode`` offers the `/execute_braketest` service which, in interaction with the ``CanOpenBraketestAdapter``,
 executes a braketest on each drive of the manipulator.
+
+## ModbusAdapterOperationModeNode
+The ``ModbusAdapterOperationModeNode`` offers the `/get_operation_mode` service for accessing the currently
```

Comment:
I would opt for a dedicated document on all these safety related features along with the general concept rather than going into detail here and there...